### PR TITLE
Updated changelog for version 2.9.0

### PR DESCRIPTION
**Pull request by ddeidda**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,42 @@
 CHANGELOG
 =========
 
+2.9.0
+=====
+
+**ENHANCEMENTS**
+
+* Added support for heterogeneous instance types with Slurm Scheduler.
```
Comment (mauri-melato) [inline CHANGELOG.rst:10]: Add support for multiple queues and multiple instance types feature with the Slurm scheduler.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,42 @@
...
+* Added CLI utility to convert ParallelCluster configuration files from single to Heterogeneous instance types.
```
Comment (mauri-melato) [inline CHANGELOG.rst:11]: Add CLI utility <command-name> to convert configuration files with Slurm scheduler to new forrmat to support multiple queue configuration.

Also move this at the bottom (or close to the bottom) of the Enh section

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,42 @@
...
+* Extended DCV support to ARM instances.
+* Added support for Torque-Slurm wrapper.
```
Comment (mauri-melato) [inline CHANGELOG.rst:13]: "Torque-Slurm wrapper"  ?!  -  Add script wrapper to support Torque-like commands with the Slurm scheduler.

File: CHANGELOG.rst [hunk 4]
```diff
@@ -2,6 +2,42 @@
...
+* Extended support for disabling hyperthreading to instances (like \*.metal) that don't support CpuOptions in launch
```
Comment (mauri-melato) [inline CHANGELOG.rst:14]: to disable... on instances (e.g. ) in LaunchTemplate

File: CHANGELOG.rst [hunk 5]
```diff
@@ -2,6 +2,42 @@
...
+  template.
+* Enable nfsv4 for computing instances file system mounting.
```
Comment (mauri-melato) [inline CHANGELOG.rst:16]: Enable support for NFS 4 for the filesystems shared from the head node.

File: CHANGELOG.rst [hunk 6]
```diff
@@ -2,6 +2,42 @@
...
+
+**CHANGES**
+
+* Introduced new configuration sections and parameters to support Heterogeneous Instance Types.
```
Comment (mauri-melato) [inline CHANGELOG.rst:20]: reword, HIT is internal jargon

File: CHANGELOG.rst [hunk 7]
```diff
@@ -2,6 +2,42 @@
...
+* Added new log streams to CloudWatch: chef-client, clustermgtd, computemgtd, slurm_resume, slurm_suspend.
+* Removed dependency on cfn-init in compute nodes bootstrap.
+* Added support for queue names in pre/post install scripts.
+* Use PAY_PER_REQUEST billing mode for DynamoDb table in govcloud regions.
+
+**BUG FIXES**
+
+* Solved dpkg lock issue with Ubuntu that prevented custom AMI creation in some cases.
+* Added/improved sanity checks for some configuration parameters.
+* Prevented ignored changes from being reported in ``pcluster update``'s output.
```
Comment (mauri-melato) [inline CHANGELOG.rst:36]: 's ?

File: CHANGELOG.rst [hunk 8]
```diff
@@ -2,6 +2,42 @@
...
+
+**CHANGES**
+
+* Introduced new configuration sections and parameters to support Heterogeneous Instance Types.
+* Optimized scaling logic with Slurm scheduler, no longer based on Auto Scaling Groups.
+* A Route53 private hosted zone is now created together with the cluster and used in DNS resolution inside cluster
+  nodes.
+* Upgrade Slurm to version 20.02.4.
+* Upgrade NICE DCV to version 2020.1-9012.
+* Use private ip instead of master node's hostname when mounting shared NFS drives.
```
Comment (mauri-melato) [inline CHANGELOG.rst:26]: 's ?

File: CHANGELOG.rst [hunk 9]
```diff
@@ -2,6 +2,42 @@
...
+* Added support for queue names in pre/post install scripts.
+* Use PAY_PER_REQUEST billing mode for DynamoDb table in govcloud regions.
+
+**BUG FIXES**
+
+* Solved dpkg lock issue with Ubuntu that prevented custom AMI creation in some cases.
+* Added/improved sanity checks for some configuration parameters.
+* Prevented ignored changes from being reported in ``pcluster update``'s output.
+* Fixed incompatibility issues with python 2.7 for ``pcluster update``.
+* Handled AMI validation error when AMI not found.
```
Comment (mauri-melato) [inline CHANGELOG.rst:38]: ?

Comment (ddeidda) [reply CHANGELOG.rst:38]: https://github.com/aws/aws-parallelcluster/commit/b339bcd52da2472883675e3700c369783071b93d

Maybe can be considered part of "Add/improve sanity checks for some configuration parameters."

File: CHANGELOG.rst [hunk 10]
```diff
@@ -2,6 +2,42 @@
...
+
+* Introduce new configuration sections and parameters to support multiple queues and multiple instance types.
+* Optimize scaling logic with Slurm scheduler, no longer based on Auto Scaling groups.
+* A Route53 private hosted zone is now created together with the cluster and used in DNS resolution inside cluster nodes
+  when using Slurm scheduler.
+* Upgrade Slurm to version 20.02.4.
+* Upgrade NICE DCV to version 2020.1-9012.
+* Use private ip instead of master node hostname when mounting shared NFS drives.
+* Add new log streams to CloudWatch: chef-client, clustermgtd, computemgtd, slurm_resume, slurm_suspend.
+* Remove dependency on cfn-init in compute nodes bootstrap.
```
Comment (enrico-usai) [inline CHANGELOG.rst:29]: Is it true for sge and torque too?

Comment (lukeseawalker) [reply CHANGELOG.rst:29]: yes

Comment (ddeidda) [reply CHANGELOG.rst:29]: It seems so to me

File: CHANGELOG.rst [hunk 11]
```diff
@@ -2,6 +2,42 @@
...
+
+**CHANGES**
+
+* Introduce new configuration sections and parameters to support multiple queues and multiple instance types.
+* Optimize scaling logic with Slurm scheduler, no longer based on Auto Scaling groups.
+* A Route53 private hosted zone is now created together with the cluster and used in DNS resolution inside cluster nodes
+  when using Slurm scheduler.
+* Upgrade Slurm to version 20.02.4.
+* Upgrade NICE DCV to version 2020.1-9012.
+* Use private ip instead of master node hostname when mounting shared NFS drives.
```
Comment (mauri-melato) [inline CHANGELOG.rst:27]: IP

File: CHANGELOG.rst (hunk 10 above)
Comment (mauri-melato) [inline CHANGELOG.rst:29]: I'd say a word about avoiding throttling  on CloudFormation and delays when a large number of compute  nodes join the cluster at once. 
And in this sense, it's an enhancement

File: CHANGELOG.rst (hunk 11 above)
Comment (mauri-melato) [inline CHANGELOG.rst:27]: Also add:
```
* Upgrade EFA installer to version 1.9.5:
  * EFA configuration: ``efa-config-1.4`` (from efa-config-1.3)
  * EFA profile: ``efa-profile-1.0.0``
  * EFA kernel module: ``efa-1.6.0`` (no change)
  * RDMA core: ``rdma-core-28.amzn0`` (no change)
  * Libfabric: ``libfabric-1.10.1amazon1.1`` (no change)
  * Open MPI: openmpi40-aws-4.0.3 (no change)
```

File: CHANGELOG.rst [hunk 12]
```diff
@@ -2,6 +2,49 @@
...
+* Remove dependency on cfn-init in compute nodes bootstrap in order to avoid throttling and delays caused by CloudFormation when a large number of compute nodes join the cluster.
+
+**CHANGES**
+
+* Introduce new configuration sections and parameters to support multiple queues and multiple instance types.
+* Optimize scaling logic with Slurm scheduler, no longer based on Auto Scaling groups.
+* A Route53 private hosted zone is now created together with the cluster and used in DNS resolution inside cluster nodes
+  when using Slurm scheduler.
+* Upgrade EFA installer to version 1.9.5:
+  * EFA configuration: `efa-config-1.4` (from efa-config-1.3)
```
Comment (mauri-melato) [inline CHANGELOG.rst:27]: double back quotes

**Review by mauri-melato — APPROVED** [review]
Please apply the comment about the double back quotes where needed and then it LGTM.

**Review by mauri-melato — APPROVED** [review]

**Pull request merged**